diff --git a/editor_legacy/authoring/project/src/ProjectBuilder.cpp b/editor_legacy/authoring/project/src/ProjectBuilder.cpp
index 8617d4267..e748eb893 100644
--- a/editor_legacy/authoring/project/src/ProjectBuilder.cpp
+++ b/editor_legacy/authoring/project/src/ProjectBuilder.cpp
@@ -5,10 +5,11 @@ namespace lux::editor
 {
     ProjectBuilder::BuildResult ProjectBuilder::build() && noexcept
     {
-        const auto valid =
-            validateProjectManifest(ProjectManifest{value_.project_id, value_.name, {}, {}, value_.plugins});
-        if (!valid)
+        auto project_manifest = ProjectManifest{value_.project_id, value_.name, {}, {}, value_.plugins};
+        const auto valid = validateProjectManifest(std::move(project_manifest));
+        if (!valid){
             return lux::cxx::unexpected(ProjectBuildFailure{EProjectBuildError::MANIFEST, valid.error()});
+        }
         if (value_.initial_scene)
         {
             const auto& initial = *value_.initial_scene;
