diff --git a/editor/application/extensions/include/lux/engine/editor/extensions/EditorExtension.hpp b/editor/application/extensions/include/lux/engine/editor/extensions/EditorExtension.hpp
index b89602a17..a86d46acb 100644
--- a/editor/application/extensions/include/lux/engine/editor/extensions/EditorExtension.hpp
+++ b/editor/application/extensions/include/lux/engine/editor/extensions/EditorExtension.hpp
@@ -5,7 +5,7 @@
 
 namespace lux::editor::extensions
 {
-    inline constexpr std::uint32_t kEditorExtensionVersion = 7;
+    inline constexpr std::uint32_t kEditorExtensionVersion = 8;
     inline constexpr const char* kEditorExtensionSymbol = "lux_editor_exports_v7";
     struct ContributionCounts final
     {
